UnityProject/Assets/Scripts/Inventory/NamedSlot.cs

```diff
@@ -35,3 +35,31 @@ public enum NamedSlot
 	none = 2048,
 
 }
+
+// NOTE: Ensure that NamedSlotFlagged == 2^(NamedSlot).
+/// <summary>
+/// All possible item slots as a flagged enum.
+/// </summary>
+[Serializable, Flags]
+public enum NamedSlotFlagged
+{
+	None = 0,
+	Outerwear = 1,
+	Belt = 2,
+	Head = 4,
+	Feet = 8,
+	Mask = 32,
+	Uniform = 64,
+	LeftHand = 128,
+	RightHand = 256,
+	Eyes = 512,
+	Back = 1024,
+	Hands = 2048,
+	Ear = 4096,
+	Neck = 8192,
+	Handcuffs = 16384,
+	ID = 32768,
+	Storage01 = 65536,
+	storage02 = 131072,
+	SuitStorage = 262144
```

Comment:
you could do
```suggestion
None = 1 << 0,
Outerwear = 1 << 1,
Belt = 1 << 2,
Head = 1 << 3,
Feet = 1 << 4,
Mask = 1 << 5,
Uniform = 1 << 6,
LeftHand = 1 << 7,
RightHand = 1 << 8,
Eyes = 1 << 9,
Back = 1 << 10,
Hands = 1 << 11,
Ear = 1 << 12,
Neck = 1 << 13,
Handcuffs = 1 << 14,
ID = 1 << 15,
Storage01 = 1 << 16,
storage02 = 1 << 17,
SuitStorage = 1 << 18
```
and not worry about calculating 2^(NamedSlot) each time